src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -280,6 +280,20 @@ private async Task<CreationResultStatus> EnterInstallFlowAsync()
                 }
             }
 
+            Reporter.Output.WriteLine("The following template sources will be installed:");
```

Comment:
Can there be zero install requests somehow? If yes, maybe we can add "No template sources found." and then `return`.